src/libraries/System.Private.CoreLib/src/System/Globalization/CompareInfo.Icu.cs

```diff
@@ -36,6 +42,7 @@ private void IcuInitSortHandle(string interopCultureName)
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Inlining large methods like this one has negative impact on performance of other codepaths in the calling method. For example, I have a tried a quick test: `compareInfo.Compare((ReadOnlySpan<char>)"Hello world", (ReadOnlySpan<char>)"world", CompareOptions.Ordinal);`. It is about 20% slower with this change. Something to check as well.